README.md

```diff
@@ -126,27 +126,33 @@ Usage:
     <imgref>
 
 Flags:
-      --chown string           chown the ouput directory to match the specified UID:GID
-      --tls-verify             require HTTPS and verify certificates when contacting registries (default true)
-      --progress string        type of progress bar to use (e.g. verbose, term)
-      --type string            image type to build [qcow2, ami] (default "qcow2")
-      --target-arch string     architecture to build image for (default is the native architecture)
+      --chown string          chown the ouput directory to match the specified UID:GID
+      --output string         artifact output directory (default ".")
+      --progress string       type of progress bar to use (e.g. verbose,term) (default "auto")
+      --rootfs string         Root filesystem type. If not given, the default configured in the source container image is used.
+      --target-arch string    build for the given target architecture (experimental)
+      --tls-verify            require HTTPS and verify certificates when contacting registries (default true)
+      --type stringArray      image types to build [ami, anaconda-iso, gce, iso, qcow2, raw, vhd, vmdk] (default [qcow2])
+      --version               version for build
```

Comment:
```suggestion
      --version               version for bootc-image-builder
```

"version for build" is slightly confusing because I'm not sure whether it's the version of _my_ image build or the BIB image.

Reply:
Good catch, thank you! I fixed this now in the README. Fixing this in the code is a bit more work as we need to workaround cobra here.